superdesk/factory/default_settings.py

```diff
@@ -359,7 +360,15 @@ def env(variable, fallback_value=None):
 SUPERDESK_TESTING = (env('SUPERDESK_TESTING', 'false').lower() == 'true')
 
 # Default TimeZone
-DEFAULT_TIMEZONE = env('DEFAULT_TIMEZONE', 'Europe/Prague')
+DEFAULT_TIMEZONE = env('DEFAULT_TIMEZONE')
+
+if DEFAULT_TIMEZONE is None:
+    try:
```

Comment:
what I can add is a log message so the exception is not totally hidden, but settings should not fail for a timezone IMHO.